### PR TITLE
awesome-book with es6

### PR DESCRIPTION
**Pull request by frnega**
in this project added all the required files

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
@frnega - 

### This is a manual review

Kindly let me know if you don't have it working like this, if you do, kindly fix it.

Kindly compare your design to the wireframe provided.

image

Cheers and Happy coding!:clap::clap::clap:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <div class="time-div"></div>
+ 
+ <main>
+<div class="awesomebooks">
+<section class="displayed-books" id="displayed-books">
+    <h2 class="headingmain">All awesome books</h2>
+        <ul id="book-list"></ul><br>
+</section>
+<section class="add-awesombooks">
+    <!-- <hr class="line"> -->
```
[the remark this answers is not in the document]
Comment (frnega) [reply index.html]: @bellom  this is what I see it works as required
awsom-book

removed the comments both from index.html and index.css

**Comment by frnega** [comment]
@bellom  
the page works perfectly do you want a video demo I do not understand the question you asked when you click the page it goes to the desired page. you have to change to the new branch and click on GO LIVE in your visual code to see it on your browser.

list

add new


contact

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (bellom) [reply index.html]: > @bellom this is what I see it works as required awsom-book
> 
> removed the comments both from index.html and index.css

@frnega,

I appreciate that you confirmed it here, we should not be seeing two different links whenever the page is loaded, in this screenshot, I can see a list of books and also the book form. do you agree with me?

if yes, once we reload the page, let's show either the book form or the list of books but not both.

I hope this is clear now.

**Comment by frnega** [comment]
@bellom <https://github.com/bellom>

ok now I understand, I removed the book list part now only see the list of
books page.
https://github.com/frnega/awesom-book-es6new/pull/1

**Pull request merged**